mindmeld/text_preparation/tokenizers.py

```diff
@@ -101,6 +111,10 @@ def tokenize(self, text):
                 Keys include "start" (token starting index), and "text" (token text).
                 For example: [{"start": 0, "text":"hello"}]
         """
+        if text == "":
+            return []
+        if not text:
```

Comment:
why is None ever passed in?

Reply:
I don't think it is, the check is there just in case it is, should I remove it?